lib/sda/host.go

```diff
@@ -20,10 +20,11 @@ import (
 	"golang.org/x/net/context"
 )
 
-/*
-Host is the structure responsible for holding information about the current
- state
-*/
+// ExternalAPI can be used to handle external message-calls
+type ExternalAPI func(*network.Message) network.ProtocolMessage
```

Comment:
one could add it - here it just replies with a message. Then SDA takes care of sending it back:

https://github.com/dedis/cothority/blob/osdi_ssh/lib/sda/host.go#L408